airflow/contrib/executors/kubernetes_executor.py

```diff
@@ -556,15 +562,17 @@ def start(self):
 
     def execute_async(self, key, command, queue=None, executor_config=None):
         self.log.info(
-            "k8s: adding task {} with command {} with executor_config {}".format(
+            "Add task {} with command {} with executor_config {}".format(
                 key, command, executor_config
             ))
         kube_executor_config = KubernetesExecutorConfig.from_dict(executor_config)
         self.task_queue.put((key, command, kube_executor_config))
 
     def sync(self):
-        self.log.info("self.running: {}".format(self.running))
-        self.log.info("self.queued: {}".format(self.queued_tasks))
+        if self.running:
+            self.log.info("self.running: {}".format(self.running))
```

Comment:
can you use the new format if we are updating anyway?

Reply:
(everywhere :P )

Reply:
Yes